search-commons/src/main/java/no/unit/nva/search/SearchClient.java

```diff
@@ -48,11 +48,12 @@ public SearchResourcesResponse searchSingleTerm(SearchDocumentsQuery query, Stri
 
     public SearchResponse findResourcesForOrganizationIds(ViewingScope viewingScope,
                                                           int resultSize,
```

Comment:
I think the resultSize should be called internally pageSize. It makes more sense